sosww/components/dynamo_db.py

```diff
@@ -316,6 +317,72 @@ def _build_scan_iterator(self, attrs=None, table_name=None, strict=True):
         return response_iterator
 
 
+    def batch_get_items_one_table(self, keys_list, table_name=None, max_retries=0, retry_wait_base_time=0.2):
```

Comment:
You probably wanted 1 here?
There is some confusion between `tries` and `retries`

Reply:
I try once and I want no more retries